src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -123,8 +123,8 @@ public void Crossgen(string outputPath)
             {
                 foreach (var asset in package.Assets)
                 {
-                    var subDir = CreateSubDirectory(outputPath, package, asset);
-                    var targetPath = Path.Combine(subDir.FullName, asset.FileName);
```

Comment:
Pretty close. Any folder names in the .nupkg (e.g. `lib`) are left as-is as well. For example, `LIB/Net45/Newtonsoft.Json.dll` remains as-is:

`~/.nuget/packages/newtonsoft.json/9.0.1-beta/LIB/Net45/Newtonsoft.Json.dll`.

But yes, the `sha512` path has lowercase everything (after the `outputPath` that is).